eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -39,7 +39,9 @@ stages:
       - stage: Release_${{artifact.safeName}}
```

Comment:
```
${{if and(or(eq(variables['TestPipeline'], 'true'), eq(variables['Build.Reason'], 'Manual')), eq(variables['System.TeamProject'], 'internal'))}}:
```
Lets try doing something like that and manually add TestPipeline = "true" for the template pipelines.